app/src/main/java/nerd/tuxmobil/fahrplan/congress/extensions/FragmentExtensions.kt

```diff
@@ -0,0 +1,11 @@
+package nerd.tuxmobil.fahrplan.congress.extensions
+
+import android.os.Bundle
+import androidx.core.os.bundleOf
+import androidx.fragment.app.Fragment
+
+/**
+ * Sets the given [pairs] as a [Bundle] in this [Fragment].
+ */
+fun <T : Fragment> T.withArguments(vararg pairs: Pair<String, Any?>): T =
```

Comment:
I've seen you format expression functions like this a couple of times now. Are you aware this violates Android's Kotlin style guide recommendations?

> The only time an expression function should wrap to multiple lines is when it opens a block.

https://developer.android.com/kotlin/style-guide#expression_functions

Reply:
No, I was not aware of this. I format it like this to avoid exceeding a certain line length. There is no style guide applied to the project yet.

Reply:
:check Changed in 840f0a172e1c73ab69d5e5240d4075858c7f51e4.

Reply:
Does this look better to you?

Reply:
You're the maintainer. It has to look good to you. I'm just making observations :smile: 
I can recommend [ktlint-gradle](https://github.com/JLLeitschuh/ktlint-gradle), though.

Reply:
True, but also I think it makes sense to let you as the observer resolve the conversation to give you the chance to state if your suggestion was satisfied / understood.

ktlint and co ... It is on my personal never ending TODO list.